docs/UserGuide.md

```diff
@@ -110,8 +128,15 @@ Deletes the specified student from TutorBuddy
 
 Format: `delete_student INDEX` <br>
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```

Comment:
Is the "Tip:" meant to be bolded?
or is this the intended outcome:
![image](https://user-images.githubusercontent.com/39006568/112750927-b0c30200-8ffd-11eb-942e-4fec39dd17b9.png)

Reply:
Yeap should be correct. I copied over from https://github.com/nus-cs2103-AY2021S2/tp/blob/master/docs/UserGuide.md. On github, it shows as per the picture as well, however when its in the webpage, it should looks as what we expected.

Reply:
ohhh ok nice